docs/docs/spec.md

```diff
@@ -122,6 +122,14 @@ A nullness operator is one of four values:
 -   `UNSPECIFIED`
 -   `MINUS_NULL`
 
+TODO: This is the first appearance of "operator" and there is no formal definition
+of what they are used for and what they mean.
+
+The "values" is maybe also misleading - these are type operators, not runtime values.
+
+This would also be a good place to introduce the postfix notation and contrast it
```

Comment:
Hmm, yes, I see that we use the notation in the non-normative text below, and we haven't defined it yet... and in fact don't do so _ever_? It looks like we might use the notation only in non-normative text, but that's still not great.

We should at least define it under "augmented types," and then maybe the non-normative comment here can refer readers ahead to that section. I could see defining it here instead, but users are going to have to understand the basics of augmented types before the section on "the informal meaning" below makes any sense, anyway. So we might as well define it under "augmented types," since it's the notation for, well, augmented types :)

(This is another manifestation of the problem of trying to define "nullness operator" before defining "augmented type.")